src/sass/component/_section.scss

```diff
@@ -0,0 +1,3 @@
+section {
```

Comment:
An import for this seems a little extreme. We might just have one file to keep it simple for users.